content/en/docs/concepts/scheduling-eviction/pod-eviction.md

```diff
@@ -0,0 +1,375 @@
+which defaults to `10s`.
+
+## Node conditions {#node-conditions}
+
+The kubelet reports node conditions to reflect that the node is under pressure
+because hard or soft eviction threshold is met, independent of configured grace
+periods.  
+
+The kubelet maps eviction signals to node conditions as follows: 
+
+| Node Condition    | Eviction Signal                                                                       | Description                                                                                                                  |
+|-------------------|---------------------------------------------------------------------------------------|------------------------------------------------------------------------------------------------------------------------------|
+| `MemoryPressure`  | `memory.available`                                                                    | Available memory on the node has satisfied an eviction threshold                                                             |
+| `DiskPressure`    | `nodefs.available`, `nodefs.inodesFree`, `imagefs.available`, or `imagefs.inodesFree` | Available disk space and inodes on either the node's root filesystem or image filesystem has satisfied an eviction threshold |
+| `PIDPressure`     | `pid.available`                                                                       | Available processes identifiers on the (Linux) node has fallen below an eviction threshold                                   |
+
+The kubelet updates the node conditions based on the configured 
+`--node-status-update-frequency`, which defaults to `10s`.
+
+### Node condition oscillation
+
+In some cases, nodes oscillate above and below soft eviction thresholds without
+holding for the defined grace periods. This causes the reported node condition
+to constantly switch between `true` and `false`, leading to bad eviction decisions.
+
+To protect against oscillation, you can use the `eviction-pressure-transition-period`
+flag, which controls how long the kubelet must wait before transitioning a node
+condition to a different state.
+
+## Reclaiming node level resources {#reclaim-node-resources}
+
+The kubelet tries to reclaim node-level resources before it evicts end-user pods.
+
+When a `DiskPressure` node condition is reported, the kubelet reclaims node-level
+resources based on the filesystems on the node. 
+
+### With `imagefs`
+
+If the node has a dedicated `imagefs` filesystem for container runtimes to use,
+the kubelet does the following:
+
+  * If the `nodefs` filesystem meets the eviction threshlds, the kubelet deletes
+    dead pods and containers. 
+  * If the `imagefs` filesystem meets the eviction thresholds, the kubelet
+    deletes all unused images. 
+
+### Without `imagefs`
+
+If the node only has a `nodefs` filesystem that meets eviction thresholds,
+the kubelet frees up disk space in the following order:
+
+1. Delete dead pods and containers
```

Comment:
"Garbage collect" might be more appropriate here

Reply:
Just for 1. or for both?

Reply:
ping @bobbypage